exasol/ds/sandbox/runtime/ansible/roles/entrypoint/files/entrypoint.py

```diff
@@ -80,12 +85,15 @@ def exit_on_error(rc):
     with open(logfile, "w") as f:
         p = subprocess.Popen(command_line, stdout=f, stderr=f)
 
-    # TODO: display default password
+    localhost_url = url.replace("<host>", "localhost").replace("<port>", "8888")
     success_message = cleandoc(f"""
         Server for Jupyter has been started successfully.
-        You can connect with https://<ip address>:<port>.
+
+        You can connect with {url}.
+
         If using a Docker daemon on your local machine and did forward the
```

Comment:
```suggestion
        If using a Docker daemon on your local machine and you forward the
```

Also update the example in the user guide